src/control/cmd/daos/pretty/pool.go

```diff
@@ -19,14 +20,39 @@ import (
 
 const msgNoPools = "No pools in system"
 
-func getTierNameText(tierIdx int) string {
-	switch tierIdx {
-	case int(daos.StorageMediaTypeScm):
-		return fmt.Sprintf("- Storage tier %d (SCM):", tierIdx)
-	case int(daos.StorageMediaTypeNvme):
-		return fmt.Sprintf("- Storage tier %d (NVMe):", tierIdx)
-	default:
-		return fmt.Sprintf("- Storage tier %d (unknown):", tierIdx)
+func printPoolTiers(suss []*daos.StorageUsageStats, w *txtfmt.ErrWriter, fullStats bool) {
+	mdOnSSD := false
+	for tierIdx, tierStats := range suss {
+		if tierIdx >= int(daos.StorageMediaTypeMax) {
+			tierStats.MediaType = daos.StorageMediaTypeMax // Print unknown type tier.
+		}
+
+		switch {
+		case tierIdx == 0 && tierStats.MediaType == daos.StorageMediaTypeNvme:
+			// MD-on-SSD mode.
+			// TODO: Print VOS index aggregate file size across pool as distinct from
+			//       Meta-blob aggregate size.
+			if fullStats {
```

Comment:
From my point of view the information regarding the "RAM cache" size should always be printed with a pool query  as it will probably be most of the time the limiting resource of a pool.

Reply:
Refer to https://github.com/daos-stack/daos/pull/14844/files, the only time fullStats == false is passed is when we are querying a individual targets. It will be true for any printing of pool info.